src/main/java/nurseybook/model/ModelManager.java

```diff
@@ -34,11 +34,11 @@
     /**
      * Initializes a ModelManager with the given versionedNurseyBook and userPrefs.
      */
-    public ModelManager(ReadOnlyAddressBook versionedNurseyBook, ReadOnlyUserPrefs userPrefs) {
+    public ModelManager(ReadOnlyNurseyBook versionedNurseyBook, ReadOnlyUserPrefs userPrefs) {
         super();
-        requireAllNonNull(versionedNurseyBook, userPrefs);
+        CollectionUtil.requireAllNonNull(versionedNurseyBook, userPrefs);
```

Comment:
If you have imported `CollectionUtil` at the start, will using it here in the prefix be necessary?

Recurring in lines 157, 172